pkg/executor/importer/import.go

```diff
@@ -809,9 +809,13 @@ func (p *Plan) initParameters(plan *plannercore.ImportInto) error {
 		setClause = sb.String()
 	}
 	optionMap := make(map[string]any, len(plan.Options))
+	var evalCtx expression.EvalContext
+	if plan.SCtx() != nil {
+		evalCtx = plan.SCtx().GetExprCtx().GetEvalCtx()
```

Comment:
Any risk for nil exception ？

Reply:
Yes. The `ImportInto` plan (and `Analyze` plan) didn't initialize the internal base plan, so the `SCtx()` can be `nil`.

Reply:
> I think we can rely on CI to find th problem

Yes. That's why I have added this line (or the unit test will fail).
The fact "The `ImportInto` plan (and `Analyze` plan) didn't initialize the internal base plan" is found by CI :facepalm: .